lib/AST/NameLookupRequests.cpp

```diff
@@ -264,11 +334,147 @@ void swift::simple_display(llvm::raw_ostream &out,
   simple_display(out, desc.Mod);
 }
 
+void AnyObjectLookupRequest::writeDependencySink(
+    Evaluator &eval, QualifiedLookupResult l) const {
+  auto member = std::get<1>(getStorage());
+
+  auto *reqTracker = eval.getActiveDependencyTracker();
```

Comment:
This bit of boilerplate appears in multiple places. Can we pass the reqTracker in to writeDependencySink(), and not call it at all if its nullptr?

Reply:
Ah, good point. This is an artifact of the old implementation that used to assert if you tried to write into a missing tracker so I could get complete coverage...